encrypted-media-respec.html

```diff
@@ -291,7 +291,7 @@ <h2>Definitions</h2>
 
         <dt id="decryption-key-id">Key ID</dt>
         <dd>
-          <p>A <a href="#decryption-key">key</a> is associated with a key ID that is a sequence of octets and which uniquely identifies the key.
+          <p>A <a href="#decryption-key">key</a> is associated with a <dfn class="export" data-lt="Decryption key ID">key ID</dfn> that is a sequence of octets and which uniquely identifies the key.
```

Comment:
Why inline the `<dfn>` when it could go directly to within the `<dt>`?

```html
<dt id="decryption-key-id"><dfn class="export" data-lt="Decryption key ID">Key ID</dfn></dt>
```

(Ideally, the `id` on the `<dt>` would go too, but we'll need to get rid of the `<a def-id="key-id"></a>` constructs first)

Reply:
Thanks, makes sense. Updated in https://github.com/w3c/encrypted-media/pull/524/commits/4b23f67c6960cb3beee7065987e0e04c81d41228.